res/css/views/messages/_LegacyCallEvent.pcss

```diff
@@ -68,7 +68,7 @@ limitations under the License.
             &.mx_LegacyCallEvent_rejected,
             &.mx_LegacyCallEvent_noAnswer {
                 .mx_LegacyCallEvent_type_icon::before {
-                    mask-image: url("$(res)/img/voip/declined-voice.svg");
+                    mask-image: url("$(res)/img/element-icons/call/hangup.svg");
```

Comment:
I'm going to ignore this since this particular change is in the code for the legacy 1:1 call system, which we're trying to avoid investing further time on, event though it has plenty of known technical debt. I am, however, making an attempt to use inline SVGs more in new code.